running/architecture.rst

```diff
@@ -2,3 +2,72 @@ Architecture
 ============
 
 Deconst is distributed as a set of Docker containers, deployed by an Ansible playbook.
+
+Terminology
+-----------
+
+.. glossary::
+
+  content repository
+    Version controlled repository containing markup that provides content for some subset of the
+    completed site. To begin, we'll assume that all of these are in ``git`` repositories on
+    `GitHub <github.com>`_. Most of our documentation will be in Sphinx reStructuredText, but the
+    developer blog will be in Markdown as part of a Jekyll site.
+
+  map repository
+    Version controlled repository containing plain-text documents that associate subtrees of
+    indexed content, identified by a :term:`content ID` prefix, with subtrees of :term:`paths`
+    on the presented site.
+
+  content ID
+    Unique identifier assigned to a single page of content. By convention, these are URLs that
+    join the https URL of the GitHub repository, ``tree/branch`` if "branch" is not master, and
+    the subpath of the page within the *rendered* output of the documentation build tool. Most of
+    the architecture should treat these as opaque strings. Examples:
+    this page would be ``https://github.com/deconst/deconst-docs/running/architecture`` on master,
+    or ``https://github.com/deconst/deconst-docs/tree/glossary/running/architecture`` until this
+    pull request is merged.
```

Comment:
This should just be described as the canonical url for a piece of content. It might be github, or bitbucket, or a url to any web-based tool.